ntoskrnl/ke/i386/ctxswitch.S

```diff
@@ -61,12 +61,12 @@ PUBLIC @KiSwapContextInternal@0
  * BOOLEAN FASTCALL
  * KiSwapContext(PKTHREAD CurrentThread, PKTHREAD TargetThread);
```

Comment:
This list of parameters do not correspond to the modified explanations given below.